cybersyn/scripts/mod-compatibility/picker-dollies.lua

```diff
@@ -0,0 +1,14 @@
+local lib = {}
+
+-- TODO: picker dollies may have been forked/renamed for 2.0?

```

Comment:
[Even Pickier Dollies](https://github.com/hgschmie/factorio-even-pickier-dollies/blob/main/scripts/constants.lua#L15) exposes its interface under the same name for compatibility.